src/app/personside/dialogpanel/Hurtigreferat/HurtigreferatContainer.tsx

```diff
@@ -117,10 +117,10 @@ function HurtigreferatContainer(props: Props) {
                         <HurtigreferatElement
                             key={hurtigreferat.tittel}
                             tekst={hurtigreferat}
-                            sendMelding={() => sendMelding(hurtigreferat)}
+                            sendMelding={() => handleSendMelding(hurtigreferat)}
                             spinner={
-                                isPosting(props.sendMeldingResource)
-                                    ? props.sendMeldingResource.payload.fritekst === hurtigreferat.fritekst
+                                isPosting(sendResource)
+                                    ? sendResource.payload.fritekst === hurtigreferat.fritekst
```

Comment:
Denne logikken virker merksnodig for meg. :/ 
Skal vise en spinner om `isPosting` og `sendResource.payload.fritekst === hurtigreferat.fritekst`  ??

Reply:
Hehe, ja, kanskje ikke så intuituv :-P Når du åpner hurtigreferat ser du at det er masse send-knapper, så knappene må vite om de er den riktige snappen til å vise spinner. Bruker fritekst til å sjekke dette. Eventuellt kunne man lagret i lokal state at knappen ble trykket på og dermed vise spinner. Ikke noe som ble gjort i denne PR'en forøvrig.